.openshift-ci/jobs/integration-tests/envbuilder.sh

```diff
@@ -269,7 +274,7 @@ setupGCPVM() {
 
     if [[ "$GCP_VM_TYPE" =~ ^ubuntu-os ]]; then
         installDockerOnUbuntuViaGCPSSH "$GCP_VM_USER" "$GCP_VM_NAME" "$GCP_SSH_KEY_FILE"
-    elif test "$GCP_VM_TYPE" = "rhel"; then
+    elif [[ "$GCP_VM_TYPE" =~ ^rhel ]]; then
```

Comment:
Maybe we can use a more restricting regex here, to prevent any future mishaps, but for the time being it should be fine as is, I'll let you decide
```suggestion
    elif [[ "$GCP_VM_TYPE" =~ ^rhel(-sap)?$ ]]; then
```